modules/persistent-storage-snapshots-controller-provisioner.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-snapshots.adoc
+
+[id="snapshots-controller-provisioner_{context}"]
+= External controller and provisioner
+
+The controller and provisioner provide volume snapshotting. These external components run in the cluster.
+
+There are two external components that provide volume snapshotting:
+
+External controller::
+Creates, deletes, and reports events on volume snapshots.
+External provisioner::
+Creates new `PersistentVolumes` from `VolumeSnapshots`.
+
+The external controller and provisioner services are distributed as container images and can be run in the {project-title} cluster as usual. There are also RPM versions for the controller and provisioner.
```

Comment:
Two notes on this line:

1. This should be `{product-title}` instead of `{project-title}`.
2. I don't believe there are RPM versions available any longer as of OCP 4.x.